src/main/java/co/elastic/logstash/filters/elasticintegration/EventProcessorBuilder.java

```diff
@@ -128,7 +126,6 @@ public EventProcessorBuilder() {
                 org.elasticsearch.ingest.common.UriPartsProcessor.TYPE));
         this.addProcessorsFromPlugin(IngestUserAgentPlugin::new);
         this.addProcessorsFromPlugin(RedactPlugin::new);
-        this.addProcessor(SetSecurityUserProcessor.TYPE, SetSecurityUserProcessor.Factory::new);
```

Comment:
I don't see any code references to `set_security_user` in [elastic/integrations](https://github.com/search?q=repo%3Aelastic%2Fintegrations+set_security_user&type=code), and it has never been in our published list of supported processors.

I am +1 to deleting it.